README.md

```diff
@@ -10,14 +10,24 @@ This project uses the [OpenFaaS Connector SDK](https://github.com/openfaas-incub
 
 The code is under **active development** and only suitable for early adopters. For the initial version the vCenter user credentials need to be stored in plain-text in your YAML files, but this will move to using [OpenFaaS secrets](https://docs.openfaas.com/reference/secrets/) in the next version.
 
-**Note:** Currently only a pre-built Docker image is available (based on [PR#11](https://github.com/openfaas-incubator/vcenter-connector/pull/11)) and only [VM events](https://code.vmware.com/doc/preview?id=4206#/doc/vim.event.VmEvent.html) are reportable for now.
+## Supported Events
+
+The following event types (incl. their subtypes) are supported and can be used to trigger functions:
+
+- Alarm Event, e.g. `AlarmCreatedEvent`
+- Datastore Event, e.g. `DatastoreDestroyedEvent`
+- Host Event, e.g. `HostDisconnectedEvent`
+- Resource Pool Event, e.g. `ResourcePoolCreatedEvent`
+- VM Event, e.g. `VmPoweredOnEvent`
+
+For further details and naming see the [vSphere Web Services API](https://code.vmware.com/apis/358/vsphere#/doc/vim.event.Event.html) documentation.
 
 ## Example: vCenter Tagging Function
```

Comment:
Let's break this out into a separate markdown file?

./docs/example.md

For instance.